Jenkinsfile-build

```diff
@@ -0,0 +1,68 @@
+def versionTag = "v${env.BUILD_NUMBER}"
+
+pipeline {
+    agent none
+
+    stages {
+      stage('Build keycloak') {
+        agent {
+          label 'master'
+        }
+        when {
+          expression {
+            env.BRANCH_NAME == "master"
+          }
+        }
+        stages {
+          stage('Tag Release') {
```

Comment:
I think the tagging should come after the build, because we shouldn't tag something if the build failed.

I'm looking back at the frontend project, and I think we should update it too (not urgently, but at some point). It currently pushes the tag only if the tests pass (which is good), but it should probably come after the Docker step too.

Reply:
Placed tagging as last stage in build process